modules/nw-ovn-ipsec-verification.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/about-ipsec-ovn.adoc
+
+[id="nw-ovn-ipsec-verification_{context}"]
+= Verifying that IPsec is enabled
+
+As a cluster administrator, you can verify that IPsec is enabled.
+
+.Verification
+
+. To find the names of the OVN-Kubernetes control plane pods, enter the following command:
++
+[source,terminal]
+----
+$ oc get pods -n openshift-ovn-kubernetes | grep ovnkube-master
+----
++
+.Example output
+[source,terminal]
+----
+ovnkube-master-4496s        1/1     Running   0          6h39m
+ovnkube-master-d6cht        1/1     Running   0          6h42m
+ovnkube-master-skblc        1/1     Running   0          6h51m
+ovnkube-master-vf8rf        1/1     Running   0          6h51m
+ovnkube-master-w7hjr        1/1     Running   0          6h51m
+ovnkube-master-zsk7x        1/1     Running   0          6h42m
+----
+
+. Verify that IPsec is enabled on your cluster:
++
+[source,terminal]
+----
+$ oc -n openshift-ovn-kubernetes -c nbdb rsh ovnkube-master-<XXXXX> \
+  ovn-nbctl --no-leader-only get nb_global . ipsec
+----
++
+--
+where:
+
+`<XXXXX>`:: Specifies the random sequence of letters for a pod from the previous step.
```

Comment:
<xxxxx>?

Reply:
@nalhadef, a reference to `ovnkube-master-zsk7x`, so `ovnkube-master-XXXXX` because it is random every time a pod starts.